src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        LocalDate localDateFrom = getDate(dateFrom);
+        LocalDate localDateTo = getDate(dateTo);
+
+        LocalDate [] consistencyLocalData = new LocalDate[data.length];
+        String [] consistencyEmployee = new String[data.length];
+        int [] consistencyWorkingHourDuringTheParticularDay = new int [data.length];
```

Comment:
There are not enough words in the variable name, I suggest adding a couple more for solidity